uploader.json

```diff
@@ -1930,6 +1930,17 @@
             "pattern": [],
             "symbolic_name": "ksmstate"
         },
+        {
+            "file": "/var/log/libvirt/libvirtd.log",
+            "pattern": [
+		"virDomainBlockPull:",
+		"virDomainBlockCopy:",
+		"virDomainBlockCommit:",
+		"virDomainSnapshotCreateXML:",
+		"qemuMigrationSrcNBDStorageCopyBlockdev:",
```

Comment:
Comma at the end of line is extra, please remove it, I think it is the case also for uploader.v2.json. @bfahr